container-images/mds-policy-author/index.ts

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
Pretty sure this is the wrong `index` file ... should be an Express server; not a lambda handler.